.cirrus.yml

```diff
@@ -0,0 +1,74 @@
+# Cirrus CI gives free 16.0 CPUs for open-source projects. We split that to 4:4:4.
+# https://cirrus-ci.org/faq/#are-there-any-limits
+#
+# Undocumented constraints;
+# - The maximum memory limit is 4G times the number of CPUs.
+# - The number of CPUs should be mutiple of 2.
+
+task:
+  name: Vagrant
+
+  compute_engine_instance:
+    image_project: cirrus-images
+    image: family/docker-kvm
+    platform: linux
+    nested_virtualization: true
+    cpu: 2
+    memory: 8G
+
+  env:
+    HOME: /root
+    matrix:
+      BOX: fedora/35-cloud-base
+      BOX: rockylinux/8
```

Comment:
(We could just use GCP rockylinux image without Vagrant, but that may rather complicate the script, so using Vagrant seems fine)

Reply:
Yeah. We technically could maintain images with our own dependencies to speed up the build, but I'm unsure it is worth to do.